internal/pkg/service/appsproxy/http/router.go

```diff
@@ -274,6 +297,11 @@ func (r *Router) createProvider(ctx context.Context, authProvider appconfig.Auth
 	}
 	provider.proxyConfig = proxyConfig
 
+	if authProvider.AllowedRoles != nil && len(allowedRoles) == 0 {
+		r.logger.With(attribute.String("exceptionId", exceptionID)).Warnf(ctx, `empty array of allowed roles for app "%s" "%s"`, app.ID, app.Name)
+		return provider
+	}
+
```

Comment:
I would say that it should be either an error or I would ignore it.

I suggest to ignore it, and switch type from `*[]string` to `[]string`.

Reply:
It does not make sense to me that the user sets an empty array there and warnings will come TO US.